lapis/README.md

```diff
@@ -91,6 +91,30 @@ For example:
 ./gradlew bootRun --args='--silo.url=http://localhost:8091 --lapis.databaseConfig.path=../lapis-e2e/testData/singleSegmented/testDatabaseConfig.yaml --referenceGenomeFilename=../lapis-e2e/testData/singleSegmented/reference_genomes.json  --server.port=8090'
 ```
 
+If running from scratch you need to pre-build the code!!!
+```
+./gradlew build bootRun --args='--silo.url=http://localhost:8091 --lapis.databaseConfig.path=../lapis-e2e/testData/singleSegmented/testDatabaseConfig.yaml --referenceGenomeFilename=../lapis-e2e/testData/singleSegmented/reference_genomes.json --server.port=8096'
+```
```

Comment:
Really? I just tried it: I checked out a new copy of the LAPIS repo and executed `./gradlew bootRun --args=...` and it worked. I would be quite surprised if `bootRun` didn't also compile the code.

Reply:
So I'm not an expert but bootRun "rebuilds the code 'if needed'" - I definitely had the case where I add some code (that actually broke the build) and was running the bootRun successfully on an older build without realizing - it confused me a lot hence the addition of this to the docs

Reply:
But we can change the message to `bootRun rebuilds the code as needed - if you want to ensure a fresh build first explicitly build lapis`

Reply:
I've never had that case. Are you sure that you restarted LAPIS after making changes? It doesn't have hot reloading.

Reply:
I definitely did, I restarted Lapis multiple times using bootRun and only when adding the build did it actually rebuild.

Reply:
Odd. That sounds more like bug than a behavior that we should document (at least to me).